src/url.js

```diff
@@ -135,7 +135,7 @@ export function parseUrl(url, opt_nocache) {
  * @param {boolean=} opt_addToFront
  * @return {string}
  */
-function appendParamStringToUrl(url, paramString, opt_addToFront) {
+export function appendParamStringToUrl(url, paramString, opt_addToFront) {
```

Comment:
I have relatively big concerns about exposing this method given potential for XSS. At the very least, if we want to do this, could you please massage the naming to something like:

- append**Encoded**ParamStringToUrl
- **encoded**ParamString

?